lib/constants/supported-ecosystems.ts

```diff
@@ -0,0 +1,10 @@
+export const supportedEcosystems = new Set([
```

Comment:
The scope of `supportedEcosystems` is limited to vulnerabilities only. IMHO defining this `const` in the top section of `vulnerability.ts` is more reasonable than decoupling it entirely into a separate file. There's neither practical benefit, nor is it needed elsewhere.